src/util/heap/layout/vm_layout_constants.rs

```diff
@@ -24,6 +24,7 @@ pub const LOG_BYTES_IN_CHUNK: usize = 22;
 
 /** Coarsest unit of address space allocation. */
 pub const BYTES_IN_CHUNK: usize = 1 << LOG_BYTES_IN_CHUNK;
+pub const CHUNK_MASK: usize = (1 << LOG_BYTES_IN_CHUNK) - 1;
```

Comment:
There is another `CHUNK_MASK` in `src/util/metadata/side_metadata/constants.rs` that has the same definition. Should we remove that after we introduce this?

Reply:
Thanks for pointing this out. I will remove `CHUNK_MASK` from side metadata constants.